### DISCUSSION TITLE
Com o meu trabalho remoto, vou receber em dólar canadense. Existe alguma opção para "guardar" este dinheiro daqui do brasil?

### DISCUSSION THREAD

**Discussion by AssisrMatheus** [DyegoCosta/trabalhando-remoto#193] [General]
Oi gente! Felizmente consegui um emprego no Canadá em uma empresa de Quebec. Eles vão me pagar em dólar e geralmente fazem transferência internacional.

Acontece que se não fosse por conta do corona vírus eu iria me mudar pra lá. Por conta disso, irei trabalhar remotamente até que tudo isso se acalme e então depois irei para lá.

Já que vou para lá, eu estou pensando em guardar o dinheiro que receber já na moeda local, para poder gastar com os custos de lá quando precisar me mudar. Então pensei em só converter para R$ o que eu preciso realmente para manter meus custos.

Eu pensei em usar este tipo de conta: https://transferwise.com/pt/multi-currency-account/. Mas este comentário me deixou um pouco cabreiro: https://www.reddit.com/r/investimentos/comments/fr8j5g/o_que_fazer_com_sal%C3%A1rio_em_euros/flwyt8v/.

O comentário diz basicamente:

> TransferWise tem um limite bem baixo (se não me engano 60k BRL por ano), que eles não avisam mas jajá vai te foder.
> 
> TW também ainda não tem opção PJ para brasileiros.
> 
> Deixar o dinheiro no PayPal ou TW: não faça isso. Eles podem congelar sua conta a qualquer momento por qualquer motivo e aí seu dinheiro fica lá.

Sou PJ como MEI, mas ainda não tenho certificado digital. Existe algum tipo de conta bancária que me ajude nessa situação?

O Payoneer conseguiria me ajudar?

Quais tipos de imposto teria que pagar?

**Comment by copercini** [comment]
Não sei se é bom, mas tem esse banco no canada que parece ser exatamente o que você precisa: https://startright.scotiabank.com/international-application.html

**Comment by AssisrMatheus** [comment]
@copercini Dei uma olhada e no final tem como eu escolher o país de qual irei me mudar, e o Brasil não está na lista. Então aparentemente não dá :/

> We’re sorry, online account opening is not currently available in your country.

**Comment by enyamada** [comment]
Esse limite de 60 mil reais é de transferência do Brasil para o exterior, e não o contrário.
Mas se vc for receber como PJ, esqueça o Transferwise, ele não é opção. 

Recomendaria uma olhada no Payoneer. Dá pra guardar dinheiro lá (se é seguro eu não sei), e poderia transferir para uma conta PJ no Brasil.

Para o certificado vc precisa de uma certificadora e quem pode te ajudar é o seu contador.

**Comment by AssisrMatheus** [comment]
@enyamada 
Ahh entendi, achei que era do quanto eu poderia "sacar" da minha conta TW.

Então, talvez eu não precise transferir para uma conta bancária PJ essa parte do dinheiro que traria pro Brasil, então a transfer wise pode me ajudar.

Estou dando uma olhada e conversando com o chat da Contabilizei para saber se os serviços que prestam é realmente o que estou precisando no momento.

Muito obrigado!

**Comment by drischerer** [comment]
@AssisrMatheus  é legal "receber"na sua conta PJ do brasil via pagamento de uma conta PF do transferwise no seu nome?
A contabilidade nao comentou nada sobre isso?  Quando fui fazer essa parte burocrática meu contador disse que **achava** que **talvez** nao fosse bom, mas nunca confirmou nada pra mim. Decidi ficar com o Payoneer pra evitar futuras tretas mas hoje estou com um bug na minha conta que nao permite mais fazer request pros EUA.  É de boa usar o transferwise pra depositar numa conta PJ NO br mesmo??

valeu!

**Comment by yuriploc** [comment]
@drischerer Como o @copercini comentou em outra issue (https://github.com/DyegoCosta/trabalhando-remoto/issues/101#issuecomment-675005458), não é o melhor dos mundos fazer esse tipo de operação, pois dá margem pra interpretação da Receita. Porém, muita gente faz..

**Comment by dofukuhara** [comment]
Opa, e aí @AssisrMatheus !
Você achou alguma opção em que consiga receber uma remessa em dolar canadense e deixar uma parte guardada para utilizar lá no Canadá?

**Comment by fuadsaud** [comment]
Tenho um caso bastante semelhante ao do OP e a melhor solução que encontrei até agora é usar a [conta do BS2](https://empresas.bancobs2.com.br/conta-internacional) pra receber valor em USD, fazer o câmbio pra BRL do valor necessário pra pagar os encargos no Brasil e transferir o restante para uma conta PF em USD (no próprio BS2, Wise ou Nomad, por exemplo).

Quanto à NF a ideia é declarar o valor total da invoice em USD usando o câmbio dessa conversão pra BRL mesmo, mesmo que a operação seja só de uma parte do valor.

Na declaração anual do IRPF declara o montante em USD na seção de contas no exterior.

E.g.:

Invoice: USD 10k.
Câmbio: BRL 5.
Conversão: USD 1k (pra pagar encargos).
NF emitida: BRL 50k.

Consultei meu contador sobre isso e, de acordo com ele, parece uma solução okay.

**Comment by igorlvicente** [comment]
Não pensais que vale a pena guardar em reais, já que o dólar está caro?
Digamos que no futuro, quando precisareis, o dólar abaixe para 4 reais. Vocês terão muito mais dólares se tiverem guardado reais, não?

**Comment by AssisrMatheus** [comment]
Então galera, pra mim foi muito difícil achar um contador que entendesse a situação, o contador que achei que entendeu, acabou me recomendando fazer foi:
- Usar o serviço da Payoneer
- Transferir pra minha conta PF
- Pagar o carnê leão(inclusive como dito aqui no repositório)

Ele não desconsiderou o caso de PJ, mas eu que falei pra ele que ia preferir PF, como eu planejo ir pra lá em pouco (questão de meses agora), não sei se valeria a pena abrir uma conta PJ na época (mas devido aos atrasos de covid, com o dinheiro que paguei, talvez teria sido uma boa mesmo 😑)

**Comment by lima-emanuel** [comment]
Acho que guardar em real ou em dólar é uma aposta sobre se o dólar vai subir ou descer num certo intervalo de tempo. Eu acho que até o começo de 2023 o dólar ou só sobe ou oscila em torno do 5.15 (por causa do clima político do Brasil), então eu guardaria meu dinheiro em dólar e reavaliaria a estratégia em 2023. O problema de guardar em dólar é só mesmo os custos de transformar em real tempos depois, que é maior do que transformar em real na hora do recebimento da remessa.
